styles/teletype.less

```diff
@@ -365,48 +367,13 @@
   }
 
   .SitePositionsComponent-site {
-    display: flex;
-  }
+    margin-left: @avatar-margin;
 
-  &.upper-right {
-    top: @corner-margin * 3;
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
-
-    .SitePositionsComponent-site {
-      margin-left: @avatar-margin;
-      flex-direction: column;
-      justify-content: flex-start;
-      align-items: center;
-    }
-  }
-
-  &.middle-right {
-    top: 50%;
-    transform: translateY(-50%);
-    display: flex;
-    flex-direction: column;
-    align-items: flex-end;
-
-    .SitePositionsComponent-site {
-      margin-bottom: @avatar-margin;
-      flex-direction: row;
-      align-items: center;
-    }
-  }
-
-  &.lower-right {
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
-    bottom: @corner-margin;
-
-    .SitePositionsComponent-site {
-      margin-left: @avatar-margin;
-      flex-direction: column;
-      justify-content: flex-end;
-      align-items: center;
+    // TODO Improve styling of link icon
+    .icon-link {
+      position: absolute;
+      padding-left: 28px;
+      padding-top: 28px;
```

Comment:
@simurai: I really dig the what the link icon appears in your mockup:

![](https://user-images.githubusercontent.com/2988/37055374-93307a58-214f-11e8-88fc-4836ec361cdb.png)

This pull request currently hacks in a pretty rudimentary implementation: 

![image](https://user-images.githubusercontent.com/2988/37055490-dae61f88-214f-11e8-9a89-d4b4e149b060.png)

Would you be willing to update the styling to match your mockup? You can feel free to push directly to this branch.

Reply:
Added an extra pseudo element as cover:

![teletype](https://user-images.githubusercontent.com/378023/37076696-1718ffd6-221b-11e8-877a-229fdcc10b68.gif)